lib/rubocop/cop/rails/hash_enum.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for enums written with array syntax
```

Comment:
I wonder if this should include a brief explanation of why hashes are preferable...